substrate/frame/nomination-pools/src/weights.rs

```diff
@@ -17,27 +17,26 @@
 
 //! Autogenerated weights for `pallet_nomination_pools`
 //!
-//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2023-11-23, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 32.0.0
+//! DATE: 2024-02-23, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! WORST CASE MAP SIZE: `1000000`
-//! HOSTNAME: `runner-yprdrvc7-project-674-concurrent-0`, CPU: `Intel(R) Xeon(R) CPU @ 2.60GHz`
+//! HOSTNAME: `MacBook-Pro.local`, CPU: `<UNKNOWN>`
```

Comment:
Could you run the weight calculation with the ci bot?

Reply:
does bot work from a comment from everyone or only from approved members?

Reply:
maybe @Ank4n or @liamaharon you can run the bench bot here?